Repairs/app/src/main/java/com/rooio/repairs/Equipment.kt

```diff
@@ -17,23 +32,69 @@ class Equipment : NavigationBar() {
                 ViewGroup.LayoutParams(ViewGroup.LayoutParams.WRAP_CONTENT, ViewGroup.LayoutParams.WRAP_CONTENT))
 
         //sets the action bar onto the page
-
         val actionbar_inflater = layoutInflater
         val poopView = actionbar_inflater.inflate(R.layout.action_bar, null)
```

Comment:
plz change the name of this too ................................................ oops